### DISCUSSION TITLE
Upgrading GPU, Going from 1080p to 1440p

### DISCUSSION THREAD

**Discussion by chiefballsy** [VirtualDrivers/Virtual-Display-Driver#156] [Q&A] [answered]
Hey folks. Just a quick question.

I set up the virtual display for 1080p, which matches my current monitor. But I do stream to a 4k tv, and my new GPU should be able to run games in 1440p just fine. I want to limit the amount of upscaling done by the TV. Do I need to change any settings, or can I just change in-game settings to 1440p and I'm good to go?

**Comment by bud3699** [comment, marked as answer]
tad late, but change it and you're good to go
